src/configuration/types.ts

```diff
@@ -3,8 +3,6 @@ export * from '@atb-as/config-specs/lib/mobility-operators';
 
 export type FirestoreConfigStatus = 'loading' | 'success';
 
-export type PreassignedFareProductId = string;
```

Comment:
Does anyone know why we have `export * from '@atb-as/config-specs';` in this file?
Seems to me that we could import from `@atb-as/config-specs` directly in the other files 🤔

Reply:
It is a way of doing encapsulation, making the rest of the app consumers of an internal module instead of directly exposing a third party package around the app. If it really makes sense for config-specs I'm a little unsure, but I think it is a good general pattern for third party dependencies.